02-electric/electric-hero/electric-hero.css

```diff
@@ -50,25 +73,39 @@
   background-color: var(--level3-background-color);
 }
 
+@media (min-width: 500px) {
+  .electric-hero-copy-container {
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
 
 .electric-hero-title{
   text-align: center;
   font-size: var(--title-font-size-mobile);
   font-weight: var(--title-font-weight-mobile);
 }
 
+@media (min-width: 500px) {
+  .electric-hero-title {
+    text-align: center;
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
+}
+
 .gas-section-one-paragraph-container {
-  
 }
```

Comment:
add a paragraph-container-row-gap-mobile

Even though there is one paragraph.. its future-proofing.

Reply:
done